app/src/main/java/ch/epfl/sweng/swenggolf/Config.java

```diff
@@ -1,21 +1,22 @@
 package ch.epfl.sweng.swenggolf;
 
-import ch.epfl.sweng.swenggolf.database.user.User;
+import com.google.firebase.auth.FirebaseAuth;
 
 /**
  * Class Used to know if we are running a test or not.
  */
-public class TestMode {
+public class Config {
 
-    private TestMode(){}
+    private Config() {
+    }
 
-    /**
-     * onTest must be true for tests and false otherwise.
+    /*
+     *   TEST_MODE must be true for tests and false otherwise.
```

Comment:
Wrong variable name in the comment, which is not JavaDoc (missing the 2 * at the beginning).